catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/types/CoreDataType.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.data.types;
+
+/**
+ * These are the allowed values for the attribute Core#DATATYPE.
+ */
+public enum CoreDataType {
+
+    COLLECTION("Collection"), DATASET("Dataset"), EVENT("Event"), IMAGE("Image"), SERVICE("Service"), SOFTWARE(
```

Comment:
I'm not sure about adding a validator. For example, the GmdConverter populates the DATATYPE attribute from the incoming GMD document (via field /MD_Metadata/hierarchyLevel/MD_ScopeCode/@codeListValue). I'm sure an argument could be made for mapping the GMD value to the now well-defined DATATYPE value. But that should be done under another ticket, so as to not bloat this ticket. Thoughts @jlcsmith @bdeining ?